File: 2_4/2task.swift
import Foundation

func pangram(_ sentence: String) -> Bool {
    var letters = Set<Character>()
    for ch in sentence {
        if ch.isLetter {
            letters.insert(ch)
        }
    }
    return letters.count == 26
}

var sentence = ""
print("Enter a string of lowercase Latin letters: ", terminator: "")
if let input = readLine() {
    sentence = input
}
print(pangram(sentence))

File: 2_4/2task.cs
using System;
using System.Collections.Generic;

class Program {
    static bool Pangram(string sentence) {
        HashSet<char> letters = new HashSet<char>();
        foreach (char ch in sentence) {
            if (char.IsLetter(ch)) {
                letters.Add(ch);
            }
        }
        return letters.Count == 26;
    }

    static void Main() {
        string sentence;
        Console.Write("Enter a string of lowercase Latin letters: ");
        sentence = Console.ReadLine();
        Console.WriteLine(Pangram(sentence));
    }
}